### DISCUSSION TITLE
Geomap - simple 2d coordinate map

### DISCUSSION THREAD

**Discussion by featheredtoast** [grafana/grafana#60756] [Plugins]
Hello! I'm looking for a way of using simple/custom 2d coordinate map with the geomap plugin, or at least some way of converting x,y coordinates into something long/lat.

I've found a similar request [here](https://github.com/grafana/grafana/issues/44240#issuecomment-1168818621), but it didn't get a reply. I hit the same use case where I'm looking to map x,y points on a base png layer.

Currently, I'm able to add a custom map by converting the coordinate system starting from 0,0 (lat, long) with a custom slippy map by slicing a raster image and applying it at scale for covering a 10kmx10km area. In the metrics themselves, I'm calculating and adding labels converting the x,y grid to the map area at the projected lat,long. It works, it just isn't very pretty.

Would it be possible to extend geomap plugin to support simple coordinate maps like these?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
